cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -344,7 +353,18 @@ pub enum Inst {
     XmmCmove {
         size: OperandSize, // 4 or 8
         cc: CC,
-        src: RegMem,
+        consequent: RegMem,
+        alternative: Reg,
+        dst: Writable<Reg>,
+    },
+
+    // Conditionally move values based on the logical OR of two flags.
```

Comment:
So this is

```
dst = if cc1 || cc2 { consequent } else { alternative }
```

right?

Could we maybe add that pseudocode to the comments for these variants? I think that will be very helpful for future readers.